timemachine/fe/pdb_writer.py

```diff
@@ -2,7 +2,8 @@
 
 import tempfile
 from simtk.openmm import app
-from simtk.openmm.app import PDBFile, Topology
+from simtk.openmm.app import PDBFile
+from fe.topology import SingleTopology
```

Comment:
Does this import still resolve?

Reply:
Good catch, I think it should be either `from timemachine.fe.topology` or just `from .` (using a [relative import](https://docs.python.org/3/reference/import.html#package-relative-imports))

Reply:
Fixed in [0537775](https://github.com/proteneer/timemachine/pull/596/commits/0537775d6dd1c0024c90a27954f602633770723d), using an absolute import to be consistent with existing convention